component_styles/card.css

```diff
@@ -0,0 +1,185 @@
+.card{
+    background-color: var(--secondary-text-color) ;
+    padding: 5rem;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+}
+
+
+
+.card__container{
+    background-color: white;
```

Comment:
create variable for color if you are using it multiple times

Reply:
ok